srcpkgs/hyperrogue/files/hyperrogue.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
can you try to upstream this?

Reply:
I can, but upstream has been generally unresponsive to PRs

Reply:
Nevermind, just landed 😄